uptane-standard.md

```diff
@@ -656,6 +656,10 @@ A primary downloads, verifies, and distributes the latest time, metadata and ima
 1. Send metadata to secondaries ({{send_metadata_primary}})
 1. Send images to secondaries ({{send_images_primary}})
 
+
+Please note, the subsequent sections concerning requirements for a primary, does not prohibit implementing primary-capability on an ECU which does not communicate directly with the Uptane repositories. This allows for implementations to have multiple ECUs within the vehicle performing functions equivalent to a primary.
+Regardless if multiple primary's are included within a vehicle, each secondary ECU SHALL have a single primary responsible for providing its updates.
```

Comment:
These two lines sound like they belong in deployment considerations. When possible, we should try to optimize this doc for the simple case but leave it open for extension and clarification in the deployment doc.

Reply:
Hi Patrick - 

Thanks for the comment.
The reason for adding these two lines is to provide clarification on the specification's flexibility on the interpretation of what is defined as a primary. 
Additionally, the second line adds an additional requirement, which I believe, is appropriately placed in the specifications document.

Let me know your thoughts -

Reply:
I'm still not convinced these lines belong in this doc. Perhaps we need wider discussion on this point. Do @JustinCappos or @trishankatdatadog (for example) have opinions?

If we do decide to keep them, there are some minor grammar issues to be fixed:

> Note that the subsequent sections concerning requirements for a primary do not prohibit implementing primary capabilities on an ECU which does not communicate directly with the Uptane repositories. This allows for implementations to have multiple ECUs within the vehicle performing functions equivalent to a primary. If multiple such primaries are included within a vehicle, each secondary ECU SHALL have a single primary responsible for providing its updates.

Reply:
Hi Patrick,

Once again, thank you for the comment. :)
I agree the scope/role/management of the Deployment Considerations needs to be discussed.
I believe it will be discussed during the F2F tomorrow.

I have modified the language per your recommendation in case the general consensus' direction is to leave this content in the specification.